adapters/readpeak/readpeak.go

```diff
@@ -0,0 +1,180 @@
+package readpeak
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+  
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+  
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the Readpeak adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+	  endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
```

Comment:
@readpeak-user  just a sugestion above code can be refactored as below.

```
func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
	return &adapter{
	  endpoint: config.Endpoint,
	}, nil
}

```

Reply:
Refactored. The original was copied from the docs here https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html.